### PR TITLE
Feat/single view2

### LINKED ISSUE #11: user can view details of a single world, including images and lore.

### PR DESCRIPTION
**Pull request by sdtrachsel**
# PR Template

- [x] I have ran all tests (if applicable) and they are passing. 

### Types of changes made?

- []  testing
- []  documentation
- [x]  javascript
- [x]  functionality

### What does this PR do?
Adds functionality - added the single worldview component. This included creating a NotableItem component. Basic formatting done. Added logic that toggles NotableItem image between right and left and displays info based on type.
Updates JS for - made two new files(singleWorld.js and NotableItem.js)

### Relevant Tickets?

- Finishes iteration or ticket for iteration 1. 

### Questions/Comments/Relevant Screenshots?

When reporting a bug, include:

### CODE REVIEW

File: never-ending/src/app/NotableItem/NotableItem.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import './NotableItem.css'
 
-export const NotableItem = ({name, type, lore, img, race, date, outcome, alignment, population}) => {
+export const NotableItem = ({ imagePosition, name, type, lore, img, race, date, outcome, alignment, population }) => {
```
Comment (Adam-Meza) [inline never-ending/src/app/NotableItem/NotableItem.js:4]: curious about the image position property, Im assuming this is to meet the wireframe of ging back and forth. this is kinda how i thought to do it too.

File: never-ending/src/app/SingleWorld/SingleWorld.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ export const SingleWorld = () => {
 
     <section className="single-world-view">
       <div className="world-detail-wrapper">
-        <img className="world-img" src={world.img} alt={`Image of %{world.name}`} />
+        <img className="world-img" src={world.img} alt={`Image of ${world.name}`} />
```
Comment (Adam-Meza) [inline never-ending/src/app/SingleWorld/SingleWorld.js:58]: hell yeah alt tags

**Review by Adam-Meza — APPROVED** [review]
looks good! gonna pull it down and take a closer look before stand up

File: never-ending/src/app/NotableItem/NotableItem.js (hunk 1 above)
Comment (sdtrachsel) [reply never-ending/src/app/NotableItem/NotableItem.js:4]: @Adam-Meza Yes, this is what toggles the image being on the right or the left. It utilizes the index of the array to assign either the left or right class name.

**Pull request merged**